mega.js

```diff
@@ -52,13 +52,13 @@
                 hash = '#' + getPathHash(url, path);
             }
             else if (url.indexOf('#') > -1) {
-                const nlfe = url.match(/\/\/mega\.nz\/+(embed|drop)[!#/]+([\w-]{8,11})(?:[#!](.*))?/i);
+                const nlfe = url.match(/\/\/mega\.nz\/+(embed|drop|filerequest)[!#/]+([\w-]{8,11})(?:[#!](.*))?/i);
```

Comment:
```suggestion
                const nlfe = url.match(/\/\/mega\.nz\/+(embed|drop|filerequest#)[!#/]+([\w-]{8,11})(?:[#!](.*))?/i);
```
(?)

Reply:
@diegocr - I dont think its needed? as `[!#/]+` checks if it has a pound sign. Thoughts? Can you give me a possible case where it fails.